package.json

```diff
@@ -9,6 +9,10 @@
     "url": "git://github.com/tbranyen/backbone.layoutmanager.git"
   },
   "main": "node/index",
+  "files": [
+    "backbone.layoutmanager.js",
+    "node/index.js"
```

Comment:
Yeah,the homepage (and for this project, the wiki) are definitely the best places to go. But when I'm working without an Internet connection, it's nice to be able to dig around in `node_modules` and find some extra information. Thanks!